README.md

```diff
@@ -141,16 +141,49 @@ Check out the [upgrade guide](doc/upgrade-cluster.md).
 ```
 $ helm delete pravega-bk --purge
 ```
-
-### Uninstall the Operator
-> Note that the Bookkeeper clusters managed by the Bookkeeper operator will NOT be deleted even if the operator is uninstalled.
-
-If you want to delete the Bookkeeper cluster, make sure to do it before uninstalling the operator. Also, once the Bookkeeper cluster has been deleted, make sure to check that the zookeeper metadata has been cleaned up before proceeding with the deletion of the operator. This can be confirmed with the presence of the following log message in the operator logs.
+Once the Bookkeeper cluster has been deleted, make sure to check that the zookeeper metadata has been cleaned up before proceeding with the deletion of the operator. This can be confirmed with the presence of the following log message in the operator logs.
 ```
 zookeeper metadata deleted
 ```
 
-You can then delete the operator
+In case the operator fails to delete metadata from the zookeeper pods, it will send out a `ZKMETA_CLEANUP_ERROR` event to notify the user about this failure. The user can check this event by doing `kubectl get events`. The following is the sample describe output of the event that is generated by the operator in such a case
+
+```
+Name:             ZKMETA_CLEANUP_ERROR-nn6sd
+Namespace:        default
+Labels:           app=bookkeeper-cluster
+                  bookkeeper_cluster=pravega-bk
+Annotations:      <none>
+API Version:      v1
+Event Time:       <nil>
+First Timestamp:  2020-04-27T16:53:34Z
+Involved Object:
+  API Version:   app.k8s.io/v1beta1
+  Kind:          Application
+  Name:          bookkeeper-cluster
+  Namespace:     default
+Kind:            Event
+Last Timestamp:  2020-04-27T16:53:34Z
+Message:         failed to cleanup pravega-bk metadata from zookeeper (znode path: /pravega/pravega): failed to delete zookeeper znodes for (pravega-bk): failed to connect to zookeeper: lookup zookeeper-client on 10.100.200.2:53: no such host
+Metadata:
+  Creation Timestamp:  2020-04-27T16:53:34Z
+  Generate Name:       ZKMETA_CLEANUP_ERROR-
+  Resource Version:    864342
+  Self Link:           /api/v1/namespaces/default/events/ZKMETA_CLEANUP_ERROR-nn6sd
+  UID:                 5b4c3f80-36b5-43e6-b417-7992bc309218
+Reason:                ZK Metadata Cleanup Failed
+Reporting Component:   bookkeeper-operator
+Reporting Instance:    bookkeeper-operator-6769886978-xsjx6
+Source:
+Type:    Error
+Events:  <none>
+```
+
+>In case the ZKMETA_CLEANUP_ERROR event has been generated by the operator, the user is expected to manually delete the metadata from zookeeper prior to reinstall.
```

Comment:
Irrespective of event generation, if BK log shows metadata deletion failed, user is expected to manually delete the metadata from zookeeper.

Reply:
done